Libraries/Components/TextInput/TextInput.js

```diff
@@ -1492,6 +1602,21 @@ const ExportedForwardRef: React.AbstractComponent<
       keyboardType={
         inputMode ? inputModeToKeyboardTypeMap[inputMode] : keyboardType
       }
+      autoComplete={
+        Platform.OS === 'android'
+          ? // $FlowFixMe
```

Comment:
What is this suppressing? If we do add a suppression, could we suppress the specific error?

Reply:
Hi @NickGerleman this is suppressing the `prop-missing` error because the `autoCompleteWebToAutoCompleteAndroidMap` object does not include all `autoComplete` keys/values. The reason for this is just because it would make the object super huge and we can just fallback to `autoComplete` when `autoCompleteWebToTextContentTypeMap[autoComplete]` is undefined. I've just updated the suppression to specify the `prop-missing` error tho